pkg/flow/adapter/xslttransform/adapter_test.go

```diff
@@ -256,6 +264,60 @@ func TestXSLTTransformEvents(t *testing.T) {
 	}
 }
 
+func TestXSLTTransformKSINK(t *testing.T) {
+	testCases := map[string]struct {
+		allowXSLTOverride bool
```

Comment:
Always `false` in the tests below, so it can be removed. A boolean is `false` by default.

Reply:
Marking not resolved.

Reply:
sorry! I removed the reference to this but never removed this guy.